`FAILED install/tests/layer_tests/tensorflow_tests/test_tf_BatchMatrixInverse.py::TestBatchMatrixInverse::test_batch_matrix_inverse_basic[ ie_device:CPU - precision:FP16 - adjoint:False - input_type:<class 'numpy.float32'> - input_shape:[2, 3, 3] ] - NotImplementedError: Op BatchMatrixInverse is not available in GraphDef version 1645. It has been removed in version 13. Use MatrixInverse instead.. `

--- a/tensorflow/core/api_def/base_api/api_def_BatchMatrixInverse.pbtxt
+++ b/tensorflow/core/api_def/base_api/api_def_BatchMatrixInverse.pbtxt
@@ -1,11 +1,8 @@
 op {
   graph_op_name: "BatchMatrixInverse"
-  deprecation {
-    version: 2
-    explanation: "Use tf.linalg.inv instead."
-  }
   description: <<END
-DEPRECATED: This operation is deprecated. Use tf.linalg.inv instead.
+DEPRECATED: This operation is deprecated and will be removed in a future version.
+Use tf.linalg.inv instead.
 
 Computes the inverse of one or more square invertible matrices or their
 adjoints (conjugate transposes).

